src/mnode/ticket-processor.cpp

```diff
@@ -152,10 +165,11 @@ bool CPastelTicketProcessor::ValidateIfTicketTransaction(const int nHeight, cons
     CMutableTransaction mtx(tx);
 
     std::string error_ret;
+    bool compressed;
```

Comment:
initialize to false

Reply:
thanks. this is my bad. a little bit still kept golang stuff in my head :).